packages/chat-stateful-client/src/types/ReadReceiptsBySenderId.ts

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * An array of key value pairs that stores each sender's display name and last read message
+ *
+ * @public
+ */
+export type ReadReceiptsBySenderId = { [key: string]: { lastReadMessage: string; displayName: string } };
```

Comment:
This type is not part of stateful client, the principle is statefulClient and components have their own types to keep themselves decoupled - if a type is shared between statefulClient and components then it is a stateful client type, otherwise it is only component type, move this to packages/react-components/src/types/